src/www/get_browser_webpack_config.mjs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import electronConfig from './webpack_electron.mjs';
+import cordovaConfig from './webpack_cordova.mjs';
+import {getWebpackBuildMode} from '../build/get_webpack_build_mode.mjs';
+
+/*
+  Inputs:
+  => the platform and buildMode
+
+  Outputs:
+  => the webpack config for the given platform and buildMode
+*/
+export const getBrowserWebpackConfig = (platform, buildMode) => {
+  let webpackConfig;
+
+  switch (platform) {
```

Comment:
Here is a place where doing platform.isElectron(), platform.isCordova() would be nice. No need to change for this PR though, but it's duplicate logic.

Reply:
Reiterating what I said above:

> Yeah, to be honest, I've been approaching these scripts thinking they're all going to be thrown out anyway. This is a bigger refactor we can do if we decide we want to keep them, but I'm hoping they become unnecessary once we move to a monorepo.